src/Repository/Geo/ZoneRepository.php

```diff
@@ -50,6 +50,7 @@ public function zoneableAsZone(ZoneableInterface $zoneable): Zone
 
         $zone->activate($zoneable->isActive());
         $zone->setName($zoneable->getName());
+        $zone->setGeoData($zoneable->getGeoData());
```

Comment:
It seems that `ZoneableInterface` has not `getGeoData`, same for others methods getName, getCode, isActive 🤔

Reply:
`ZoneableInterface` extends `GeoInterface`, thus they are part of the contract

Reply:
oh yes, my bad, sorry